examples/bq-email-exports/terraform/main.tf

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+provider "google" {
+  project = var.project_id
+  region  = var.region
+}
+
+resource "google_service_account" "service_account" {
+  account_id   = var.service_acct_name
+  display_name = "Service Account for BQ email exports"
+}
+
+resource "google_project_iam_binding" "iam_bq_admin" {
+  project = var.project_id
+  role    = "roles/bigquery.admin"
+
+  members = [
+    "serviceAccount:${google_service_account.service_account.email}",
+  ]
+}
+
+resource "google_project_iam_binding" "iam_storage_admin" {
+  project = var.project_id
+  role    = "roles/storage.admin"
+
+  members = [
+    "serviceAccount:${google_service_account.service_account.email}",
```

Comment:
nit: Since members are constant, it might be more DRY to for_each over a list of roles.